packages/git/node-tests/writer-test.js

```diff
@@ -111,7 +111,7 @@ describe('git/writer', function() {
 
   describe('create', function() {
     it('saves attributes', async function () {
-      let { data:record } = await writers.create('master', env.session, 'articles', {
+      let { data:record } = await writers.create(env.session, 'articles', {
```

Comment:
@robustdj this is an example of a writer change. we need to remove the branch parameter from any `writers.create()` calls being made (I think there is some middleware in one of the cards that is doing this).